readme.md

```diff
@@ -155,7 +161,7 @@ touch 'new file' # updates the file's access and modification timestamp if it al
 ```shell
 mkdir 'untitled folder'
 # or
-mkdir -p 'path/may/not/exist/untitled folder'
+mkdir -p 'path/may/not/exist/untitled\ folder'
```

Comment:
I'm not sure why this change is needed?

Reply:
Good to get in the habit of escaping spaces.